guardian/manage-frontend
# Removing non serviceable countries from address dropdown

## What does this change?
Removing countries we are no longer able to service from country address drop downs. This includes the address filed where you can update your delivery address and correspondence address for all products.



## How to test
Check country drop downs in the Settings -> correspondence address OR on a deliverable product update your address


## How can we measure success?



## Have we considered potential risks?



## Images



## Accessibility
